ppr-ui/src/composables/fees/FeeSummary.vue

```diff
@@ -132,6 +132,29 @@
             ${{ feeSummary.serviceFee.toFixed(2) }}
           </div>
         </li>
+        <li
+          v-else-if="hasProcessingFee && feeSummary.processingFee > 0"
+          id="processing-fee-summary"
+          :class="[$style['fee-container'], $style['fee-list__item'], 'pb-4', 'pr-4', 'py-4']"
+        >
+          <div :class="$style['fee-list__item-name']">
+            Staff Processing Fee
+          </div>
+          <div :class="$style['fee-list__item-value']">
+            ${{ feeSummary.processingFee.toFixed(2) }}
```

Comment:
Hmm not sure about `toFixed()` here. It will round the number and could cause some unexpected results. I know this is not new, but do you know why we need to use it?

Reply:
It just applies the decimals, if i had to guess. 
ie i don't believe `$25.00` is the value applied, it's probably `$25`. 
My guess is that it was to satisfy design, not necessarily do any rounding.